### PR TITLE
Upgrade ruby to 2.6.3

### PR DESCRIPTION
**Pull request by marcomarcianesi**

### CODE REVIEW

**Comment by marcomarcianesi** [comment]
[upgrade projects to ruby 2.6.3](https://trello.com/c/kVdAI7St/578-upgrade-projects-to-ruby-263)

File: Dockerfile [hunk 1]
```diff
@@ -1,27 +1,74 @@
-FROM kyero/ruby
+FROM ruby:2.6.3
```
Comment (ivopt) [inline Dockerfile:1]: Could this perhaps be a descendent of our `docker-ruby` image?

---

I honestly don't know the full implications of ☝️.. just a hypothetical question

Comment (marcomarcianesi) [reply Dockerfile:1]: what does it change?

**Review by ivopt — APPROVED** [review]
👌

**Pull request merged**